### ISSUE TITLE
Whole gutter background changed when gitsigns numhl enabled

### ISSUE THREAD

**Issue by rylee-randall** [ellisonleao/gruvbox.nvim#230]
**Describe the bug**
When you enable gitsigns' numhl feature the entire gutter for the changed lines has its background colour changed producing ugly grey blocks.

**To Reproduce**
Install  gitsigns and gruvbox.nvim, enable numhl in gitsigns with `:Gitsigns toggle_numhl`, make some changes in a git tracked file.

**Expected behavior**
The line numbers of changed lines get highlighted as appropriate but the background colour stays the same.

**Screenshots**
Here is what it looks like when numhl is enabled, not the big grey blocks in the gutter for every line that has the number highlighted for changes
![image](https://user-images.githubusercontent.com/99856880/229402130-4a1fea95-a3de-4771-befa-f0f272da9e19.png)
Here is the lack of the big grey blocks when numhl is disabled
![image](https://user-images.githubusercontent.com/99856880/229402148-1465be45-7bea-4ccd-91be-b0ff61e20bb6.png)

**Comment by mrtnvgr** [comment]
Does this solve your issue?

```lua
require("gruvbox").setup({
	overrides = {
		SignColumn = { link = "Normal" },
		GruvboxGreenSign = { bg = "" },
		GruvboxOrangeSign = { bg = "" },
		GruvboxPurpleSign = { bg = "" },
		GruvboxYellowSign = { bg = "" },
		GruvboxRedSign = { bg = "" },
		GruvboxBlueSign = { bg = "" },
		GruvboxAquaSign = { bg = "" },
	},
})
```

**Comment by rylee-randall** [comment]
Yup that seems to resolve it thankyou! Would you mind explaining what this does and why its required?

**Comment by mrtnvgr** [comment]
> Yup that seems to resolve it thankyou! Would you mind explaining what this does and why its required?

This snippet removes background from sign column and signs.
